deploy-board/deploy_board/webapp/templatetags/utils.py

```diff
@@ -653,8 +657,12 @@ def hostButton(host):
 
 @register.filter("hostIcon")
 def hostIcon(host):
-    if host['state'] == 'PROVISIONED':
-        return 'fa-refresh fa-spin'
+    if host["state"] == "PROVISIONED":
+        duration = (time.time() * 1000 - host["lastUpdateDate"]) / 1000
+        isStale = duration > DEFAULT_STALE_THRESHOLD
```

Comment:
Same staleness check we do for agents (5 mins)

Reply:
some cloud init operation can run for >5mins, you might need to take a higher threshold here like 15 mins. stale for agent is different than here.